### DISCUSSION TITLE
Expanded Detail Panel in Headless Mode

### DISCUSSION THREAD

**Discussion by ronkristoff** [KevinVandy/material-react-table#1027] [Q&A]
Hi, do you have any example of how to make details panel using headless. I tried the headless example and add the configurations for the detail panel, the icon appears but when clicked the information for the expanded panel is not showing.

**Comment by KevinVandy** [comment]
The "examples" for anything in headless mode is the MRT source code
Headless mode is pretty much just you forking, or rebuilding the table components that MRT already has
like, you should be able to find everything you need here, right? https://github.com/KevinVandy/material-react-table/blob/v2/packages/material-react-table/src/components/body/MRT_TableBodyRow.tsx

Also, the TanStack Table examples in those docs are a great place to start too
Here's the most basic detail panel/sub-components example: https://tanstack.com/table/latest/docs/framework/react/examples/sub-components
